force-app/main/default/objects/External_Committee__c/listViews/All.listView-meta.xml

```diff
@@ -11,4 +11,7 @@
         <value>TODAY</value>
     </filters>
     <label>My DBAAC Meetings</label>
+    <sharedTo>
+        <allInternalUsers></allInternalUsers>
```

Comment:
@deepakmulamalla Why is the list view sharing settings updated in this PR? It does not look like it was previously shared with internal users. Was this a requirement to share it with internal users? Please confirm.

Reply:
It's done as part of ESA-1965 and the list view is restricted to internal users.